File: Iniciante/URI 2930.pas
var
    e : int64;
    d : int64;

begin
    readLn(e, d);    

    if (e > d) then
        writeln('Eu odeio a professora!')
    else if ((d - e) >= 3) then
        writeln('Muito bem! Apresenta antes do Natal!')
    else if ((e + 2) < 24) then
        writeln('Parece o trabalho do meu filho!'#10'TCC Apresentado!')
    else
        writeln('Parece o trabalho do meu filho!'#10'Fail! Entao eh nataaaaal!')
end.

File: Iniciante/URI 2930.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val leitor = Scanner(System.`in`);
    val e: Int = leitor.nextInt();
    val d: Int = leitor.nextInt();
    if (e > d) {
        println("Eu odeio a professora!");
    } else if ((d - e) >= 3) {
        println("Muito bem! Apresenta antes do Natal!");
    } else if ((e + 2) < 24) {
        println("Parece o trabalho do meu filho!\nTCC Apresentado!");
    } else {
        println("Parece o trabalho do meu filho!\nFail! Entao eh nataaaaal!");
    }
}